type-definitions/flow-tests/record.js

```diff
@@ -170,23 +169,6 @@ person.get('unknown');
 // $FlowExpectedError[prop-missing]
 person.set('unknown', 'Thales');
 
-// Note: not <TPerson>
```

Comment:
I'm not sure what the use case here is (and `PersonRecord` is no longer parameterized anyway) so I removed the test altogether.  If someone can explain the use case I can perhaps update it instead. :)